extensions/wikia/WikiaBar/models/WikiaBarDataFailsafeModel.class.php

```diff
@@ -30,71 +30,71 @@ class WikiaBarDataFailsafeModel extends WikiaBarModelBase {
 			'button-1-class=cup
 button-1-text=New In Entertainment
-button-1-href=http://www.wikia.com/Entertainment
+button-1-href=//www.wikia.com/Entertainment
 button-2-class=travel
 button-2-text=Fall TV
-button-2-href=http://www.wikia.com/Entertainment/TV_Schedule
+button-2-href=//www.wikia.com/Entertainment/TV_Schedule
 button-3-class=beauty
 button-3-text=@WikiaEnt
 button-3-href=https://twitter.com/wikiaent
 line-1-text=Go behind the scenes for the 23rd Bond film.
-line-1-href=http://jamesbond.wikia.com/wiki/Skyfall
+line-1-href=//jamesbond.wikia.com/wiki/Skyfall
 line-2-text=Learn about the mysterious blackout in J.J. Abrams\' Revolution.
-line-2-href=http://revolution.wikia.com/wiki/Revolution_Wiki
+line-2-href=//revolution.wikia.com/wiki/Revolution_Wiki
 line-3-text=Explore the final adaptation of Twilight: Breaking Dawn
-line-3-href=http://twilightsaga.wikia.com/wiki/Breaking_Dawn
+line-3-href=//twilightsaga.wikia.com/wiki/Breaking_Dawn
 line-4-text=Look up the Lyrics of your favorite song now!
-line-4-href=http://lyrics.wikia.com/Lyrics_Wiki
+line-4-href=//lyrics.wikia.com/Lyrics_Wiki
 line-5-text=Follow the development process of the new Hobbit trilogy.
-line-5-href=http://lotr.wikia.com/wiki/The_Hobbit_%28films%29
+line-5-href=//lotr.wikia.com/wiki/The_Hobbit_%28films%29
 ',
 			WikiFactoryHub::CATEGORY_ID_LIFESTYLE => 'button-1-class=cup
 button-1-text=Gaming: New Releases
-button-1-href=http://www.wikia.com/Video_Games
+button-1-href=//www.wikia.com/Video_Games
 button-2-class=travel
 button-2-text=New in Entertainment
-button-2-href=http://www.wikia.com/Entertainment
+button-2-href=//www.wikia.com/Entertainment
 button-3-class=beauty
 button-3-text=Lifestyle: Travel
-button-3-href=http://travel.wikia.com/wiki/Wikia_Travel
+button-3-href=//travel.wikia.com/wiki/Wikia_Travel
 line-1-text=The world\'s most popular logos at your fingertips.
-line-1-href=http://logos.wikia.com/wiki/Main_Page
+line-1-href=//logos.wikia.com/wiki/Main_Page
 line-2-text=Your family\'s genealogy awaits you.
-line-2-href=http://familypedia.wikia.com/wiki/Family_History_and_Genealogy_Wiki
+line-2-href=//familypedia.wikia.com/wiki/Family_History_and_Genealogy_Wiki
 line-3-text=You have questions. We have answers.
-line-3-href=http://how-to.wikia.com/wiki/How_To
+line-3-href=//how-to.wikia.com/wiki/How_To
 line-4-text=Recipes for all your favorite dishes.
-line-4-href=http://recipes.wikia.com/wiki/Recipes_Wiki
+line-4-href=//recipes.wikia.com/wiki/Recipes_Wiki
 line-5-text=Over 8,000 pages dedicated to cars.
-line-5-href=http://automobile.wikia.com/wiki/Autopedia
+line-5-href=//automobile.wikia.com/wiki/Autopedia
 '
 		),
 		'de' => array(
 			WikiFactoryHub::CATEGORY_ID_GAMING =>
 			'button-1-class=cup
 button-1-text=Videospiele
-button-1-href=http://de.wikia.com/Videospiele
+button-1-href=//de.wikia.com/Videospiele
 button-2-class=cup
 button-2-text=Herbstauswahl
 button-2-href=http://de.community.wikia.com/wiki/Benutzer_Blog:Foppes/Herbstauswahl_2012
```

Comment:
We leave this our because of the language prefix. Right?

Reply:
Yep, these can't support HTTPS yet, so making them relative would break the links if you saw them on say de.wikia.com which would support HTTPS.